### PR TITLE
[New] add `SquashPrTitleUsedAsDefault` metric

### LINKED ISSUE #84: add metric: "default to PR titles on squashmerge"
See https://github.blog/changelog/2022-05-11-default-to-pr-titles-for-squash-merge-commit-messages/

### PR DESCRIPTION
**Pull request by thehanimo**
Closes #84 

Not sure about the permissions required for `squashPrTitleUsedAsDefault`

### CODE REVIEW

**Review by ljharb — APPROVED** [review]
LGTM! just the one question

File: config/metrics.js [hunk 1]
```diff
@@ -95,6 +95,10 @@ module.exports = {
 		extract: (item) => item.deleteBranchOnMerge,
 		permissions: ['ADMIN'],
 	},
+	SquashPrTitleUsedAsDefault: {
+		extract: (item) => item.squashPrTitleUsedAsDefault,
+		permissions: ['ADMIN'],
```
[the remark this answers is not in the document]
Comment (thehanimo) [reply config/metrics.js:100]: do you have MAINTAIN access to any repo? if you do, can you check if you can access this setting?

**Comment by thehanimo** [comment]
Why did the Automatic Rebase action fail when I was still able to rebase the PR using 'Update branch' on here? 🤔

**Comment by ljharb** [comment]
lol good question

**Pull request merged**